src/Eshopworld.WorkerProcess/Stores/CosmosDbLeaseStore.cs

```diff
@@ -38,27 +39,34 @@ public class CosmosDbLeaseStore : ILeaseStore
         public async Task InitialiseAsync()
         {
             await _documentClient.CreateDatabaseIfNotExistsAsync(new Database {Id = _options.Value.Database})
-                .ConfigureAwait(false);
-
+                    .ConfigureAwait(false);
+            
             DocumentCollection collectionDefinition = new DocumentCollection
             {
                 Id = _options.Value.Collection
             };
 
             collectionDefinition.PartitionKey.Paths.Add("/leaseType");
 
+            var leaseConstraint = new UniqueKey
+            {
+                Paths = new Collection<string> { "/leaseType" }
+            };
+
+            collectionDefinition.UniqueKeyPolicy.UniqueKeys.Add(leaseConstraint);
+
             await _documentClient.CreateDocumentCollectionIfNotExistsAsync(
                 UriFactory.CreateDatabaseUri(_options.Value.Database),
                 collectionDefinition,
-                new RequestOptions {OfferThroughput = _options.Value.OfferThroughput}).ConfigureAwait(false);
+                new RequestOptions { OfferThroughput = _options.Value.OfferThroughput }).ConfigureAwait(false);
```

Comment:
Can you change your VS settings so it does not modify formatting if code is not changed?